04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -6,8 +6,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "__author__ = 'Mike Fitzpatrick <mfitzpatrick@noao.edu>, Alice Jacques <alice.jacques@noao.edu>, NOIRLab Astro Data Lab Team <datalab@noao.edu>'\n",
-    "__version__ = '20200922'\n",
+    "__author__ = 'Mike Fitzpatrick <fitz@noirlab.edu>, Alice Jacques <alice.jacques@noirlab.edu>, NOIRLab Astro Data Lab Team <datalab@noirlab.edu>'\n",
```

Comment:
Same comment as in PR77; the short alias emails likely don't work yet.

Reply:
Corrected